modules/dnn/src/layers/max_unpooling_layer.cpp

```diff
@@ -110,17 +111,12 @@ class MaxUnpoolLayerImpl CV_FINAL : public MaxUnpoolLayer
                     int index = idxptr[i_wh];
                     if (!(0 <= index && index < outPlaneTotal))
                     {
-                        std::cerr
-                            << "i_n=" << i_n << std::endl
-                            << "i_c=" << i_c << std::endl
-                            << "i_wh=" << i_wh << std::endl
-                            << "index=" << index << std::endl
-                            << "maxval=" << inptr[i_wh] << std::endl
-                            << "outPlaneTotal=" << outPlaneTotal << std::endl
-                            << "input.size=" << input.size << std::endl
-                            << "indices.size=" << indices.size << std::endl
-                            << "outBlob=" << outBlob.size << std::endl
-                            ;
+                        CV_LOG_DEBUG(NULL, cv::format(
```

Comment:
Debug logger messages are completely turned off in release builds by default (in compile time - no way to enable them in runtime).

It is detail message for the error below.

Reply:
You are right! Replaced with CV_LOG_ERROR.